diff -burNp main/launchpadbugs/utils.py main-dud/launchpadbugs/utils.py
--- main/launchpadbugs/utils.py	2009-01-08 12:25:24.000000000 +0100
+++ main-dud/launchpadbugs/utils.py	2009-03-14 17:16:29.580431490 +0100
@@ -4,10 +4,14 @@
 import os
 import re
 import subprocess
-import apt_pkg
 import libxml2
 import urlparse
 
+try:
+    import apt_pkg
+except ImportError, e:
+    pass
+
 from exceptions import LaunchpadURLError, PythonLaunchpadBugsValueError
 from lpconstants import BASEURL
 
@@ -40,10 +44,10 @@ def find_version_number(show_nick=False)
 
     # We're using a package
     if path.startswith('/usr'):
-        output = subprocess.Popen(["dpkg-query", "-W", "python-launchpad-bugs"], 
+	output = subprocess.Popen(["rpm", "-q",	"python-launchpad-bugs", "--qf", "%{V}-%{R}"],
                                    stdout=subprocess.PIPE).communicate()[0]
         try:
-            return output.split()[1]
+            return output
         except:
             # not installed as a package
             return "unknown"
